src/Provider/Vkontakte.php

```diff
@@ -1,51 +1,78 @@
 <?php
-/*!
-* Hybridauth
-* https://hybridauth.github.io | https://github.com/hybridauth/hybridauth
-*  (c) 2017 Hybridauth authors | https://hybridauth.github.io/license.html
-*/
 
 namespace Hybridauth\Provider;
 
 use Hybridauth\Adapter\OAuth2;
 use Hybridauth\Exception\UnexpectedApiResponseException;
-use Hybridauth\Data;
-use Hybridauth\User;
+use Hybridauth\Data\Collection;
+use Hybridauth\User\Profile;
 
 /**
- * Vkontakte OAuth2 provider adapter.
+ * Vkontakte provider adapter.
+ *
+ * Example:
+ *
+ *   $config = [
+ *       'callback'  => Hybridauth\HttpClient\Util::getCurrentUrl(),
+ *       'keys'      => ['id' => '', 'secret' => ''],
+ *   ];
+ *
+ *   $adapter = new Hybridauth\Provider\Vkontakte($config);
+ *
+ *   try {
+ *       if (!$adapter->isConnected()) {
+ *           $adapter->authenticate();
+ *       }
+ *
+ *       $userProfile = $adapter->getUserProfile();
+ *   }
+ *   catch(\Exception $e) {
+ *       print $e->getMessage() ;
+ *   }
  */
 class Vkontakte extends OAuth2
 {
     /**
-    * {@inheritdoc}
-    */
+     * {@inheritdoc}
+     */
     protected $apiBaseUrl = 'https://api.vk.com/method/';
 
     /**
-    * {@inheritdoc}
-    */
-    protected $authorizeUrl = 'http://api.vk.com/oauth/authorize';
+     * {@inheritdoc}
+     */
+    protected $authorizeUrl = 'https://api.vk.com/oauth/authorize';
 
     /**
-    * {@inheritdoc}
-    */
+     * {@inheritdoc}
+     */
     protected $accessTokenUrl = 'https://api.vk.com/oauth/token';
 
     /**
-    * {@inheritdoc}
-    */
-    protected $scope = 'email';
+     * {@inheritdoc}
+     */
+    protected $scope = 'email,offline';
```

Comment:
As new change, I've added new scope, for getting access_token without expiration, because VK API doesn't provide mechanism for refreshing only access token.

They are suggest to [re-authenticate](https://vk.com/dev/authcode_flow_user?f=4.%20Receiving%20access_token):

> Get a new access_token every time it expires or if a user changes their login or password or deletes an application from the list. 

We can't refresh token, we need to get new one and this is bad solution, because we need to redirect user to external page and getting there code for exchanging on token, during processing another request:

Token validation is called inside OAuth2::apiRequest
```php
        // refresh tokens if needed
        if ($this->hasAccessTokenExpired() === true) {
            $this->refreshAccessToken();
        }
```
What that mean? For example, my token is expired and i want to get user profile, I'm loading adapter and calling getUserProfile method, where token will be validated and with suggested flow from API, we need to re-authenticate user (redirect to page to get new code/token), that's mean that getUserProfile wouldn't be executed